packages/devtools_app/lib/src/shared/screen.dart

```diff
@@ -56,11 +55,9 @@ enum ScreenMetaData {
     title: 'Memory',
     icon: Octicons.package,
     requiresDartVm: true,
-    // ignore: avoid_redundant_argument_values, false positive
-    requiresConnection: !FeatureFlags.memoryOffline,
+    requiresConnection: false,
     tutorialVideoTimestamp: '?t=420',
-    // ignore: avoid_redundant_argument_values, false positive
-    worksWithOfflineData: FeatureFlags.memoryOffline,
+    worksWithOfflineData: true,
```

Comment:
and then leave everything that was previously checking the `FeatureFlags.memoryOffline` flag